javascript/app.js

```diff
@@ -245,6 +237,11 @@ cocktailApp.revealButton.addEventListener("click", function () {
 
 cocktailApp.searchButton.addEventListener("click", function (e) {
     e.preventDefault();
+
+    cocktailApp.fadeOut(cocktailApp.recipeContainer, 5);
+    cocktailApp.fadeOut(cocktailApp.ingredientList, 5);
+    cocktailApp.fadeOut(cocktailApp.instructionList, 5);
+
     const searchTerm = cocktailApp.searchInput.value;
```

Comment:
I think for 2 word searches (like bourbon sour) we need to run
const something = searchTerm.replaceAll(" ","_");
so that the url can take the search term
(replace spaces: bourbon_sour)

Reply:
done! good call :)